include/dlaf/permutations/general.h

```diff
@@ -89,6 +89,7 @@ void permute(comm::CommunicatorGrid grid, common::Pipeline<comm::Communicator>&
 
   DLAF_ASSERT(i_begin >= 0 && i_begin <= i_end, i_begin, i_end);
 
+  // TODO: assert revision (PR#854)
```

Comment:
Presumably this TODO means you want to wait for #854 to be merged before this PR?

Reply:
We will synchronise with @rasolca, but an idea might be that I rebase #854 and make it a fork of this one (so we merge that one in this one).

Reply:
My idea was to merge this PR and when ready #854. But I'm fine with @albestro suggestion as well.